devbuild.sh

```diff
@@ -228,10 +237,11 @@ if [ "${VERBOSE}" = true ]; then
   MAKE_SETTINGS="${MAKE_SETTINGS} VERBOSE=1"
 fi
 
-# source the environment file for this platform/compiler combination, then build the code
-printf "... Source ENV_FILE and create BUILD directory ...\n"
-module use ${SRC_DIR}/env
-. ${ENV_FILE}
+# source the module file for this platform/compiler combination, then build the code
+printf "... Load MODULE_FILE and create BUILD directory ...\n"
+module use ${SRC_DIR}/modulefiles
+module purge
```

Comment:
I think that a purge here was causing some issues on Cheyenne in the past. @mkavulich Can you confirm that is still problematic.